README.md

```diff
@@ -92,7 +125,7 @@ class QuoteStartedTest extends TestCase
         $this->json('POST', '/api/quote', ['email' => 'quote@start.com']);
 
         // Assert.
-        $this->assertEventStoreEventRaised('quote_started', 'quotes', ['email' => 'quote@start.com']);
+        $this->assertEventStoreEventRaised('QuoteStarted', 'quotes', ['email' => 'quote@start.com']);
```

Comment:
Think we should use generics, as:

```suggestion
        $this->assertEventStoreEventRaised('event.name', 'stream_name', ['email' => 'foo@bar.com']);
```

Using `QuoteStarted` here gives an impression that it's the same thing as within EventServiceProvider::$listen (which isn't the case)